IPython/terminal/ptutils.py

```diff
@@ -12,8 +12,12 @@
 
 class IPythonPTCompleter(Completer):
     """Adaptor to provide IPython completions to prompt_toolkit"""
-    def __init__(self, ipy_completer):
-        self.ipy_completer = ipy_completer
+    def __init__(self, shell):
```

Comment:
Not every internal utility class is a public API, so I have no qualms changing this one. I can make it backward-compatible, though.